clients/web/src/views/widgets/HierarchyWidget.js

```diff
@@ -435,6 +435,16 @@ var HierarchyWidget = View.extend({
                 });
             }, this)
         };
+        if (type === 'collection') {
+            params = _.extend({
+                name: this.parentModel.escape('name'),
+                additionalText: '<b>' + this.parentModel.escape('name') + '</b>' +
+                                ' contains <b>' + this.parentModel.escape('nFolders') +
+                                ' folders</b> taking up <b>' +
+                                formatSize(this.parentModel.escape('size')) + '</b>',
+                msgConfirmation: true
```

Comment:
If the size and `nFolders` are both 0, I think we should skip the confirmation check.